src/rpc/handlers/AccountInfo.hpp

```diff
@@ -131,8 +132,11 @@ class AccountInfoHandler {
         static auto const rpcSpecV1 = RpcSpec{
             {JS(account), validation::AccountValidator},
             {JS(ident), validation::AccountValidator},
+            {JS(ident), check::Deprecated{}},
             {JS(ledger_hash), validation::Uint256HexStringValidator},
-            {JS(ledger_index), validation::LedgerIndexValidator}
+            {JS(ledger_index), validation::LedgerIndexValidator},
+            {JS(ledger), check::Deprecated{}},
+            {JS(strict), check::Deprecated{}}
```

Comment:
If we add warning for "strict" here, we shall add it for other handlers too. I remember a lot of handlers have this deprecated field.